src/reactions/__tests__/aggregateReactions-test.js

```diff
@@ -1,6 +1,7 @@
 import deepFreeze from 'deep-freeze';
 
 import aggregateReactions from '../aggregateReactions';
+import { NULL_USER } from '../../nullObjects';
```

Comment:
Maybe just omit deactivated users?

Reply:
I guess, for v1 we can go with this. And afterward we can handle  `deactivated users` case. :)